docs/platforms/unreal/configuration/setup-crashreporter/index.mdx

```diff
@@ -174,3 +174,11 @@ are subject to future change and defined currently as:
 ## Issues With the Crash Reporter
 
 Check out this community-generated [thread in Sentry's forum](https://forum.sentry.io/t/unreal-engine-crash-reporter-cant-get-it-to-work/7643) for UE Crash Reporter troubleshooting tips and solutions.
+
+## Store Minidumps as Attachments
```

Comment:
Also sometimes we display different headings, for example, under `data-management` and I did not want to have "Store Minidumps As Attachments" 2x. Suggestions?